apps/admin-portal/src/api/identity-provider.ts

```diff
@@ -19,13 +19,18 @@
 import { AxiosHttpClient } from "@wso2is/http";
 import { GlobalConfig, ServiceResourcesEndpoint } from "../configs";
 import {
+    ApplicationTemplateListInterface,
     FederatedAuthenticatorListItemInterface,
     FederatedAuthenticatorMetaInterface,
     HttpMethods,
     IdentityProviderInterface,
     IdentityProviderListResponseInterface,
-    IdentityProviderResponseInterface
+    IdentityProviderResponseInterface, IdentityProviderTemplateListInterface, IdentityProviderTemplateListItemInterface
 } from "../models";
+import {AxiosError, AxiosResponse} from "axios";
```

Comment:
Add a space between the import curly braces.

Reply:
Fixed with a71ec75bd7ff9034c30a22f8fd552a5d61bc892e.